x/profiles/keeper/common_test.go

```diff
@@ -140,6 +145,11 @@ func (suite *KeeperTestSuite) SetupTest() {
 		scopedIBCKeeper,
 	)
 
+	suite.sk = subspaceskeeper.NewKeeper(
+		suite.storeKey,
```

Comment:
It should be keys[subspacetypes.StoreKey], or subspacetypes.StoreKey should not be created in NewKVStoreKeys.